cpp/Brewfile

```diff
@@ -25,8 +25,7 @@ brew "flatbuffers"
 brew "git"
 brew "glog"
 brew "grpc"
-brew "llvm"
-brew "llvm@8"
```

Comment:
We need LLVM 8 for Clang Tools such as `clang-tidy`.

Reply:
@kou how about separating the Brewfile required for CI and for local development?

Reply:
It's OK to me.
But I'm not sure we can maintain both of them.